react/README.md

```diff
@@ -1,14 +1,21 @@
 # React Tech Guides
+At [Wolox](https://wolox.co), we use our own version of [create-react-app](https://github.com/Wolox/wolox-react-scripts#creating-an-app) to create our React projects. Check it out to find a detailed guide to start with your project.
```

Comment:
This is not true anymore :). To create our projects, we now use our [react-bootstrap](https://github.com/Wolox/react-bootstrap). This is detailed below, so I wouldn't put this here.

Reply:
good to know lul